README.md

```diff
@@ -49,6 +49,25 @@ jail.create("11.1-RELEASE")
 
 Libiocage comes bundles with a CLI tool called `ioc`. It is inspired by the command line interface of [iocage](https://github.com/iocage/iocage) but meant to be developed along with the library and to spike on new features.
 
+### Custom Release (e.g. running -CURRENT)
+
+#### Initially create release dataset
+
+```sh
+zfs create zroot/iocage/releases/custom/root
```

Comment:
how about storing a configuration in `zroot/iocage/releases/custom/config.json` ?
that details what this release is cut from (in case we need to `freebsd-update` it) and which directories need to be mounted into a jail that's based on it.

Reply:
good call!

Reply:
This feature in not a replacement for templates, but rather a way to run unsupported releases, e.g. -STABLE or -CURRENT. For these installations freebsd-update is not possible.
Updates are simply done by writing the binaries and libraries over the release, not touching the config files, which are written by make distribution.
This approach is used by ezjail as well.